ui/app/components/toolbar-filters.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * @module ToolbarFilters
+ * `ToolbarActions` components are containers for Toolbar filters and toggles.
```

Comment:
Do we want to mention that this should always be used inside of a `Toolbar`? (also for the other components).

Reply:
+1 to what Matthew said -- I think that note would be helpful. It also looks like this is the description for `ToolbarActions` instead of `Toolbar Filters`.